sdks/php/templates/api.mustache

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * {{classname}}
- * PHP version {{phpVersion}}
+ * PHP version 7.4
```

Comment:
Should we be hardcoding this? Seems useful to keep the template variable

Reply:
Comes from upstream, I'm fine keeping it as-is